### DISCUSSION TITLE
How to use LORA in Runpod and Paperspace?

### DISCUSSION THREAD

**Discussion by fishonainternet** [TheLastBen/fast-stable-diffusion#2327] [Q&A]
Hi i have used Google Colab (before you have to pay) for Stable Diffusion. I'm trying to move to either Runpod or Paperspace, but i like to know if you can use LORA in those platforms or not? Because for Colab you just have to upload it in Google Drive.

**Comment by TheLastBen** [comment]
same as colab, use the file browser panel in the left to upload the files to the right folders

**Reply by fishonainternet** [reply]
Do i have to upload LORA & Models over and over again each time i connected to a different pod?

**Reply by TheLastBen** [reply]
use the save cell to backup your loras to huggingface, use your huggingface token the restore them whenever you start a new pod
